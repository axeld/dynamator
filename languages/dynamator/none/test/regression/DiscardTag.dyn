<dynamator language="none">
  <class name="discard_tag">
    <discard-tag/>
  </class>
</dynamator>
