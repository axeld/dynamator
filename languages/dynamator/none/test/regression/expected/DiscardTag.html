<html>
  <head>
    <title>Test Discard Tag</title>
  </head>
  <body>
    discard this tag
    discard this tag
    discard this tag
    discard this tag
    discard this tag
    discard this tag
  </body>
</html>
<!-- generated by Dynamator Sun Feb 15 17:23:23 CST 2004 -->
